pkg/controller/ingress/create.go

```diff
@@ -379,6 +379,10 @@ func (lbc *EngressController) createLoadBalancerSvc() error {
 			Selector: labelsFor(lbc.Config.Name),
 		},
 	}
+	if lbc.Options.ProviderName == "aws" {
+		// ref: https://github.com/kubernetes/kubernetes/blob/release-1.5/pkg/cloudprovider/providers/aws/aws.go#L79
+		svc.Annotations["service.beta.kubernetes.io/aws-load-balancer-proxy-protocol"] = ""
```

Comment:
`svc.Annotations["service.beta.kubernetes.io/aws-load-balancer-proxy-protocol"] = "*"`

Value should be star. `Right now we only accept the
// value "*" which means enable the proxy protocol on all ELB backends`